docs/ViewModel.md

```diff
@@ -99,99 +69,132 @@ Component.extend({
 ```
 
 If this component HTML was inserted into the page like:
-
-    var template = stache("<my-paginate/>")
-    $("body").append(template())
-
+```js
+var template = stache("<my-paginate/>")
+var frag = template();
+document.body.appendChild(frag);
+```
 It would result in:
 
     <my-paginate>Page 1</my-paginate>
 
-This is because the provided viewModel object is used to extend a [can-map] like:
-
-    CustomMap = Map.extend({
-      offset: 0,
-      limit: 20,
-      page: function(){
-        return Math.floor(this.attr('offset') / this.attr('limit')) + 1;
-      }
-    })
+This is because the provided ViewModel object is used to create an instance of [can-define/map/map] like:
+```js
+CustomMap = DefineMap.extend({
+	offset: {value: 0},
+	limit: {value: 20},
+	page: function(){
+		return Math.floor(this.offset / this.limit) + 1;
+	}
+})
+```
 
-Any primitives found on a `Map`'s prototype (ex: `offset: 0`) are used as
+Any properties found on a `DefineMap`'s prototype with a value (ex: `offset: {value: 0}`) are used as
```

Comment:
The [can-define.types.value] property definition makes `offset` default to `0` and `limit` default to `20`.